src/spec/page-objects/template-page.js

```diff
@@ -0,0 +1,121 @@
+class TemplatePage {
+  constructor() {
+    this._templateNameLabel = $("form .templates--information .information-fields label[for='uName'");
+    this._templateFromEmailLabel = $("form .templates--information .information-fields label[for='uFromEmail'");
+    this._templateFromNameLabel = $("form .templates--information .information-fields label[for='uFromName'");
+    this._templateSubjectLabel = $("form .templates--information .information-fields label[for='uSubject'");
```

Comment:
Is it not so much specific? for example, if _templates--information_ is the form's class,  it should be not necessary using _information-fields_, right?

Reply:
Yes it is not necessary, I will remove it to be more flexible.